advanced/sr_gui_cyberglove_calibrator/src/sr_gui_cyberglove_calibrator/cyberglove_calibrer.py

```diff
@@ -317,7 +317,7 @@ def write_calibration_file(self, filepath):
         # write the text to a file
         try:
 
-            output = open(filepath[0], "w")
+            output = open(filepath, "w")
```

Comment:
I changed this to be a real filepath instead of qt_related tuple also containing the filter, which the calibrer class does not need